Library/MockRemoteConfigClient.swift

```diff
@@ -3,6 +3,8 @@ import Foundation
 
 private class MockRemoteConfigValue: RemoteConfigValue {
   var bool = false
+
+  override var boolValue: Bool { self.bool }
```

Comment:
Even though this change needs to be made, it is more relevant to my creator dashboard ff related [changes ](https://github.com/kickstarter/ios-oss/pull/1828/files#diff-6cdfce6a5b1a0951b6d5a3cbf5a74312978f2f18581e39a79355841b9fec8db4). Might be worth removing to maintain a clearer context

Reply:
yea good call out, I noticed that too but some tests need to pass using the `MockRemoteConfigClient` and under the hood it uses the `RemoteConfigValue` which in our helper classes references `.boolValue`.

Reply:
ah right. ok cool